controllers/admin/AdminStatusesController.php

```diff
@@ -64,7 +64,7 @@ public function init()
     /**
      * init all variables to render the order status list.
      */
-    protected function initOrderStatutsList()
```

Comment:
@taoufiqaitali Hi, could you create an another protected method `initOrderStatutsList` which call `initOrderStatusList` for removing the BC Break ? Thanks

Reply:
@Progi1984 ok done, i created function that call initOrderStatusList